includes/wf_crm_webform_postprocess.inc

```diff
@@ -1642,6 +1712,39 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
     return round($this->totalContribution, 2);
   }
 
+ /**
+   * Calculate pro rated membership fee.
+   * @param Integer $type
+   * @param Float $price
+   * @param Integer $c
+   * @param Array $item
+   * @return Float $price
+   */
+  private function pro_rated_membership_fee($type, $price, $c, $n, $item) {
```

Comment:
can you replace $c and $n and $item with names that represent what they actually about.